tools/cldr-apps/src/main/java/org/unicode/cldr/web/WebContext.java

```diff
@@ -1538,43 +1405,43 @@ public void setSession() {
 
         logger.fine("Session Now=" + session + ", user=" + user);
 
-        // guest?
-        if (UserRegistry.userIsTC(user)) {
-            // allow in administrator or TC.
-        } else if ((user != null) && (session == null)) { // user trying to log in-
-            if (CookieSession.tooManyUsers()) {
-                System.err.println(
-                        "Refused login for "
-                                + email
-                                + " from "
-                                + userIP()
-                                + " - too many users ( "
-                                + CookieSession.getUserCount()
-                                + ")");
-                setSessionMessage(
-                        "We are swamped with about "
-                                + CookieSession.getUserCount()
-                                + " people using the SurveyTool right now! Please try back in a little while.");
-                return;
-            }
-        } else if (session == null || (session.user == null)) { // guest user
-            if (CookieSession.tooManyObservers()) {
-                if (session != null) {
+        // allow in administrator or TC.
+        if (!UserRegistry.userIsTC(user)) {
```

Comment:
this is an auto-refactor by IntelliJ, to fix a warning about empty "if" block -- it comes out looking more complicated than it really is

Reply:
makes sense.   Though, It'd be easier to read if the refactor was a separate PR or at least a separate commit.

Reply:
Oh, this IS the refactor PR.  I reread it.  It looks like it has some wording cleanup too, which is fine.  LGTM!